src/api/spec/models/staging/workflow_spec.rb

```diff
@@ -83,21 +80,27 @@
       it { expect(subject).to be_empty }
     end
 
-    context 'with requests and some are in staging projects and some not' do
-      let!(:bs_request_2) do
+    context 'with request in state review whether they are in staging projects or not' do
+      let(:bs_request_2) do
         create(:bs_request_with_submit_action,
+               state: :review,
                target_project: project.name,
                target_package: target_package.name,
                source_project: source_project.name,
                source_package: source_package.name)
       end
 
       before do
-        bs_request.staging_project = staging_project
-        bs_request.save
+        bs_request
+        bs_request_2.change_review_state(:accepted, by_group: group.title)
+        # A review by project is created by default. See BsRequest#apply_default_reviewers. We have to accept it too.
```

Comment:
Create the source_package with :as_submission_source then this doesn't happen

Reply:
Good tip! :heart: Thanks!